service/handlers.go

```diff
@@ -202,6 +203,17 @@ func (h handler) createWorkflowFromGit(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// Read from ddb and continue execution even if it fails
```

Comment:
this is the general pattern for handler func calls into dbb. attempts to use ddb, but should never return an error back to the actual api client, just log it.